Library/Homebrew/formula.rb

```diff
@@ -593,7 +593,11 @@ def oldnames
   # @api internal
   sig { returns(T::Array[String]) }
   def aliases
```

Comment:
The new def is kinda clunky, but matches the style of the preceding def (`oldnames`). 🤷